src/main/java/org/jenkinsci/plugins/jsch/JSchConnector.java

```diff
@@ -0,0 +1,54 @@
+package org.jenkinsci.plugins.jsch;
+
+import com.cloudbees.jenkins.plugins.sshcredentials.SSHAuthenticatorException;import com.jcraft.jsch.JSch;
+import com.jcraft.jsch.JSchException;
+import com.jcraft.jsch.Session;
+
+/**
+ * @author stephenc
+ * @since 25/10/2012 15:14
```

Comment:
🐜 Wrong Javadoc. At least since if you copy-pasted the code from somewhere.
CC @stephenc